backend/database/migrations/2022_03_09_105302_add_word_to_words_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('words', function (Blueprint $table) {
+            $table->string('word');
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('words', function (Blueprint $table) {
+            //
```

Comment:
drop the column `word` because when running `php artisan migrate:rollback` this will cause an error